specutils/tests/test_loaders.py

```diff
@@ -368,3 +376,56 @@ def test_tabular_fits_header(tmpdir):
     assert 'OBSDATE' not in hdulist[0].header
     assert 'OBSDATE' not in hdulist[1].header
     hdulist.close()
+
+
+@pytest.mark.remote_data
+def test_apstar_loader():
+    '''Test remote read and automatic recognition of apStar spec from URL.
+    '''
+    apstar_url = ("https://data.sdss.org/sas/dr16/apogee/spectro/redux/r12/"
+                  "stars/apo25m/N7789/apStar-r12-2M00005414+5522241.fits")
+    spec = Spectrum1D.read(apstar_url)
+
+    assert isinstance(spec, Spectrum1D)
+    assert spec.flux.size > 0
+    assert spec.uncertainty.array.min() >= 0.0
+
+
+@pytest.mark.remote_data
+def test_apvisit_loader():
+    '''Test remote read and automatic recognition of apvisit spec from URL.
+    '''
+    apvisit_url = ("https://data.sdss.org/sas/dr16/apogee/spectro/redux/r12/"
+                   "visit/apo25m/N7789/5094/55874/"
+                   "apVisit-r12-5094-55874-123.fits")
+    spec = Spectrum1D.read(apvisit_url)
+
+    assert isinstance(spec, Spectrum1D)
+    assert spec.flux.size > 0
+    assert spec.uncertainty.array.min() >= 0.0
+
+
+@pytest.mark.remote_data
+def test_aspcapstar_loader():
+    '''Test remote read and automatic recognition of aspcapStar spec from URL.
+    '''
+    aspcap_url = ("https://data.sdss.org/sas/dr16/apogee/spectro/aspcap/r12/"
+                  "l33/apo25m/N7789/aspcapStar-r12-2M00005414+5522241.fits")
+    spec = Spectrum1D.read(aspcap_url)
+
+    assert isinstance(spec, Spectrum1D)
+    assert spec.flux.size > 0
+    assert spec.uncertainty.array.min() >= 0.0
+
+
+@pytest.mark.remote_data
+def test_muscles_loader():
+    '''Test remote read and automatic recognition of muscles spec from URL.
```

Comment:
```suggestion
    """Test remote read and automatic recognition of muscles spec from URL.
```

Reply:
Change all existing docstrings to double quotes as well, I assume?

Reply:
Yep! 😁